### ISSUE TITLE
Bug: Thread 'tokio-runtime-worker' panicked at 'Storage root must match that calculated.'

### ISSUE THREAD

**Issue by Jason2d2** [Manta-Network/Manta#663]
2022-07-10 12:56:08 [Parachain] Block prepare storage changes error: Error at calling runtime api: Execution failed: Execution aborted due to trap: wasm trap: wasm `unreachable` instruction executed
WASM backtrace:

    0: 0x2b4845 - <unknown>!rust_begin_unwind
    1: 0x27374b - <unknown>!core::panicking::panic_fmt::h5118e89563022e7e
    2: 0x212648 - <unknown>!frame_executive::Executive<System,Block,Context,UnsignedValidator,AllPalletsWithSystem,COnRuntimeUpgrade>::execute_block::h0ab5e79109062ca6
    3: 0x3e451 - <unknown>!Core_execute_block

2022-07-10 12:56:08 [Parachain] 💔 Error importing block 0xb4492992feae423a24a47b9317629dd53841316477b16a1bc4783b8d1a7b161b: consensus error: Import failed: Error at calling runtime api: Execution failed: Execution aborted due to trap: wasm trap: wasm `unreachable` instruction executed
WASM backtrace:

    0: 0x2b4845 - <unknown>!rust_begin_unwind
    1: 0x27374b - <unknown>!core::panicking::panic_fmt::h5118e89563022e7e
    2: 0x212648 - <unknown>!frame_executive::Executive<System,Block,Context,UnsignedValidator,AllPalletsWithSystem,COnRuntimeUpgrade>::execute_block::h0ab5e79109062ca6
    3: 0x3e451 - <unknown>!Core_execute_block

Hash: given=19391ac733b0537e6846399efb50b69ca95afe18aa50b4bf23c03c9792753623, expected=963090364cde38006fba030a77590efa19b48b9d783c07994f6b17dc49662f62

====================

Version: 3.2.0-2477840-x86_64-linux-gnu

   0: <unknown>
   1: <unknown>
   2: <unknown>
   3: <unknown>
   4: <unknown>
   5: <unknown>
   6: <unknown>
   7: <unknown>
   8: <unknown>
   9: <unknown>
  10: <unknown>
  11: <unknown>
  12: <unknown>
  13: <unknown>
  14: <unknown>
  15: <unknown>
  16: <unknown>
  17: <unknown>
  18: <unknown>
  19: <unknown>
  20: <unknown>
  21: <unknown>
  22: <unknown>
  23: start_thread
  24: clone

Thread 'tokio-runtime-worker' panicked at 'Storage root must match that calculated.', /home/runner/.cargo/git/checkouts/substrate-7e08433d4c370a21/fc3fd07/frame/executive/src/lib.rs:488

Thread 'tokio-runtime-worker' panicked at 'Storage root must match that calculated.', /home/runner/.cargo/git/checkouts/substrate-7e08433d4c370a21/fc3fd07/frame/executive/src/lib.rs:488

**Comment by Dengjianping** [comment]
Thanks for reporting the problem to us.

Could you provide more context from your environment?
Such as:
```
OS:
Which manta release:
Testnet or mainnet:
Did you try it again after crash?:
The command you start the node:
```
It really can help us to debug the problem.
Thanks!

**Comment by Jason2d2** [comment]
Ubuntu 20.4

Calamari

Yes, reboot, sync was successful. Node, calamari-bitManna.

Docker: --rpc-cors all --collator --rpc-methods=safe -- --telemetry-url 'wss://api.telemetry.manta.systems/submit/ 0'

**Comment by Dengjianping** [comment]
Glad you found a temporary workaround.

This is a upstream bug, and a known issue: https://github.com/paritytech/substrate/issues/9697
And there's a PR to fix this issue: https://github.com/paritytech/substrate/pull/11407

Hope they will fix it soon.

**Comment by Dengjianping** [comment]
I will keep this issue open until the bug is fixed.

**Comment by Garandor** [comment]
resolves with polkadot-v0.9.29
